src/utils/openCloseComponent.ts

```diff
@@ -60,6 +61,26 @@ function transitionEnd(event: TransitionEvent): void {
     this.open ? this.onOpen() : this.onClose();
   }
 }
+
+/**
+ * Helper to determine globally set transition duration setting.
+ *
+ * @param component
+ */
+export function onToggleUnanimatedComponent(component: OpenCloseComponent): void {
+  readTask((): void => {
+    if (component.transitionEl) {
+      const tokens: string[] = getComputedStyle(component.transitionEl).transition.split(" ");
```

Comment:
Let's rename this to better express what it's storing. I also think you can skip the type since TS will infer from the array's return type. This last comment applies to other variable declarations.